.github/workflows/workflow.yml

```diff
@@ -76,10 +79,84 @@ jobs:
         run: |
           bash tests/test.sh
 
-  publish:
-    if: ${{ github.ref == 'master' && github.event.action != 'pull_request'}}
-    needs: build
+  publish-docs:
```

Comment:
This section should be reviewed for publishing docs. Commands tested locally and on my fork (publishes correctly)